src/radical/pilot/configs/resource_ornl.json

```diff
@@ -97,8 +97,8 @@
         "lrms"                        : "TORQUE",
         "agent_type"                  : "multicore",
         "agent_config"                : "cray",
-        "agent_scheduler"             : "CONTINUOUS",
-        "agent_spawner"               : "POPEN",
+        "agent_scheduler"             : "HOMBRE",
```

Comment:
Isn't `HOMBRE` use case specific? Or it is a requirement on Titan to use homogeneous tasks in the sense of resource requests?

I do not believe that using a scheduler that is so specific as the default is the best way to go. I would suggest that either offer some other configuration, or change the use cases that really needed it change it programmatically.

Reply:
Thanks, fixed.